### ISSUE TITLE
install fails if /proc mounted in image

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#132]
**Reported by efocht on 31 Aug 2005 10:42 UTC**
```
Sometimes /proc remains mounted in the image. It just 
happened to me now with RHEL4-ia64 and a freshly 
created image. When this happens, the rsync command 
will fail because it tries to sync virtual files 
from /proc which are long gone. 
 
I am not sure about the place where this should be 
fixed. There is nothing which _has_ to be executed 
when (re-)installing a node, the closest thing is the 
starting of the si_monitord. 
 
```

**Comment by dikim33** [comment]
**Modified by jparpail on 10 Apr 2007 09:29 UTC**

**Comment by dikim33** [comment]
**Modified by jparpail on 11 Apr 2007 11:43 UTC**

**Comment by dikim33** [comment]
**Comment by jparpail on 20 Jul 2007 15:00 UTC**
We won't have 5.0.1, so mark it for 5.1

**Comment by dikim33** [comment]
**Modified by valleegr on 30 Jun 2008 06:29 UTC**

**Comment by dikim33** [comment]
**Modified by valleegr on 11 Feb 2009 19:32 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 1 Apr 2009 10:13 UTC**
We should check that during the post image creation actions.

**Comment by dikim33** [comment]
**Comment by valleegr on 4 Jun 2009 18:16 UTC**
(In [8547]) This check-in fixes #134 - We make sure /proc is umounted after the creation 
of the image.
